src/tools/importPlanGoals.js

```diff
@@ -66,32 +79,35 @@ export default async function importGoals(fileKey, region) {
     const cleanRoleTopics = [];
     const cleanGrantGoals = [];
     const cleanTopicGoals = [];
-    const currentGoals = [];
 
     await prePopulateRoles();
 
     for await (const el of grantees) {
-      let currentGranteeId;
-      let grants;
       let currentGrants = [];
-      let currentGoalName;
-      let currentGoal = {};
+      const currentGoals = [];
+      let currentGoalName = '';
+      let currentGoalNum = 0;
 
       for await (const key of Object.keys(el)) {
         if (key && (key.trim().startsWith('Grantee (distinct') || key.trim().startsWith('Grantee Name'))) {
-          grants = el[key] ? el[key].split('|')[1].trim() : 'Unknown Grant';
-          currentGrants = grants.split(',');
+          currentGrants = parseGrantNumbers(el[key]);
```

Comment:
Curious what grant numbers needed refactoring. (no need to post here though because of PII)

Reply:
There ended up being a bunch in the format "Name | Number\nName | Number" which would be expected for a multi-select column but these were in a single select form. 🤷🏻